_includes/header.html

```diff
@@ -1,17 +1,17 @@
-<a class="content-skip" href="#content">Skip to content</a>
+    <a id="content-skip" href="#content" tabindex="1">Skip to content</a>
 <header>
     <script src="/js/header.js"></script>
-    <a href="/"><img id="logo" src="/assets/img/fist.svg"></a>
+    <a href="/"><img id="logo" src="/assets/img/fist.svg" alt="A geometric red fist clenching a wrench."></a>
```

Comment:
I would say "Team 1418's logo" here or something to that effect, but this is fine also.